src/main/java/friendlyBotDeadline.java

```diff
@@ -0,0 +1,14 @@
+public class friendlyBotDeadline extends friendlyBotTask {
+
```

Comment:
Should this space be removed to be consistent with other classes (i.e friendlyBotEvent.java, friendlyBotTask.java)?

Reply:
I've edited this, thank you for the advice! :)